src/pages/dev/amplifier/chain-integration/relay-messages.mdx

```diff
@@ -0,0 +1,157 @@
+export DESTINATION_CHAIN="ethereum-sepolia"
+export DESTINATION_ADDRESS="0x8f8dedd09E23E22E1555e9D2C25D7c7332291919"
+export SOURCE_ADDRESS="0x0a3b8dc7706c47b6dd87d771df63875b1c5cd867"
+export PAYLOAD_HASH="220f68445e3cec114bff50cd6b251e3deabc7684b10280c2116b20bcc6795a96"
+export RPC="http://devnet-verifiers.axelar.dev:26657"
+```
+
+```bash
+axelard tx wasm execute $SOURCE_CHAIN_GATEWAY \
+    '{
+       "route_messages":[
+            {
+                "cc_id": {
+                    "chain":"'"$CHAIN_NAME"'",
+                    "id":"'"$TX_HASH-$TX_EVENT"'"
+                },
+                "destination_chain":"'"$DESTINATION_CHAIN"'",
+                "destination_address":"'"$DESTINATION_ADDRESS"'",
+                "source_address":"'"$SOURCE_ADDRESS"'",
+                "payload_hash":"'"$PAYLOAD_HASH"'"
+            }
+       ]
+    }' \
+    --keyring-backend test  \
+    --from wallet  \
+    --gas auto --gas-adjustment 1.5 --gas-prices 0.007uverifiers  \
+    --chain-id=devnet-verifiers \
+    --node $RPC
+```
+
+## Construct a proof
+
+[`construct_proof`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/contracts/multisig-prover/src/msg.rs#L61) takes a vector of `CrossChainID`s and builds a proof that includes the messages to be routed so that the message can be relayed to the destination gateway. It also passes the message from the gateway to the prover.
+
+```bash
+export DESTINATION_CHAIN_MULTISIG_PROVER="axelar1xz4cya4qm2ws6nzperhvc40wdjcq4872fl6d3j2s4cytyx8j80eqenv87g"
+export CHAIN_NAME="avalanche"
+export TX_HASH="0x41d08bd627d7be301e0858312b63b3ede840d7fad145a9da333c26adf5d98614"
+export TX_EVENT="0"
+export RPC="http://devnet-verifiers.axelar.dev:26657"
+```
+
+```bash
+axelard tx wasm execute $DESTINATION_CHAIN_MULTISIG_PROVER \
+    '{
+        "construct_proof":
+            {
+                "message_ids": [
+                    {
+                        "chain":"'"$CHAIN_NAME"'",
+                        "id":"'"$TX_HASH-$TX_EVENT"'"
+                    }
+                ]
+            }
+    }' \
+    --keyring-backend test \
+    --from wallet \
+    --gas auto --gas-adjustment 1.5 --gas-prices 0.007uverifiers \
+    --chain-id devnet-verifiers \
+    --node $RPC
+```
+
+## Get the proof
+
+[`get_proof`](https://github.com/axelarnetwork/axelar-amplifier/blob/f76dc8a775b8bb529cf50147bab9259d1ec7f784/contracts/multisig-prover/src/msg.rs#L81) returns the fully signed proof from the multisig prover after verifiers vote.
+
+```bash
+export DESTINATION_CHAIN_MULTISIG_PROVER="axelar1xz4cya4qm2ws6nzperhvc40wdjcq4872fl6d3j2s4cytyx8j80eqenv87g"
+export MULTISIG_SESSION_ID="3457"
+export RPC="http://devnet-verifiers.axelar.dev:26657"
+```
+
+```bash
+axelard q wasm contract-state smart $DESTINATION_CHAIN_MULTISIG_PROVER \
+    '{
+        "get_proof":{
+            "multisig_session_id":"'"$MULTISIG_SESSION_ID"'"
+        }
+    }' \
+    --node $RPC
+```
```

Comment:
Should we show the cast command or a similar instructions on how to take the output with the proof and send it to the destination chain? You can do it with `cast` (from foundry), or with metamask directly.

Reply:
How would one do it with MetaMask directly? I'll add a tabbed section.